packages/e2e-tests/plugins/block-icons/index.js

```diff
@@ -55,7 +55,7 @@
 	} );
 
 	registerBlockType( 'test/test-dash-icon', {
-		title: 'TestDashIcon',
+		title: 'TestSimpleDashIcon',
```

Comment:
This change prevents name collision. The `exact` match wasn't enough for some reason 🤷‍♂️